api/src/main/java/uk/gov/dvsa/mot/persist/TradeReadDao.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.dvsa.mot.persist;
+
+import uk.gov.dvsa.mot.trade.api.Vehicle;
+
+import java.util.Date;
+import java.util.List;
+
+public interface TradeReadDao {
+    //  List<DisplayMotTestItem> getMotHistoryByDateRange( Date startDate, Date endDate ) ;
```

Comment:
Very good question. It's something we didn't look at doing this time round. Can speak with Matthew about this.